docs/pages/x/api/date-pickers/date-calendar.json

```diff
@@ -1,5 +1,6 @@
 {
   "props": {
+    "autoFocus": { "type": { "name": "bool" } },
```

Comment:
`DateCalendar` already had this prop
No idea why the doc gen was not getting it

Same for `DateRangeCalendar` and all the pickers that use them